.travis/install_openssl_1_1_1.sh

```diff
@@ -27,22 +27,15 @@ fi
 
 BUILD_DIR=$1
 INSTALL_DIR=$2
-PLATFORM=$3
 RELEASE=1_1_1g
```

Comment:
I also had to manually change this line to 1.1.1i . Any opposition to making this a parameter to the script and having the codebuild jobs explicitly provide an arg?

Reply:
will do!